tfx_bsl/beam/run_inference.py

```diff
@@ -96,18 +98,137 @@ class OperationType(object):
 
 
 @beam.ptransform_fn
-@beam.typehints.with_input_types(Union[tf.train.Example,
-                                       tf.train.SequenceExample])
+@beam.typehints.with_input_types(tf.train.Example)
 @beam.typehints.with_output_types(prediction_log_pb2.PredictionLog)
-def RunInferenceImpl(  # pylint: disable=invalid-name
+def RunInferenceOnExamples(  # pylint: disable=invalid-name
```

Comment:
Let's use the first option of public API here to have a polymorphic RunInference and RunInferenceImpl.